vitoriapinheiro/Delta-odonto
# Feature/journey section

## What I did
1. Created the new Journey sections, made it responsive 
2. Resolved console errors
3. fixed responsiveness bugs on the whole project
4. Updated the logo
5. Upgradet the footer

## How to test

1. Run the project (yarn on the front, npm on the back)
2. Go to your local host 3001
3. Compare the Journey section with the new mockup
4. Check if it's responsive
5. Check I the rest of the site is responsive

## Technical debts
1. The rest of the tests :sob:
